packages/babel-plugin-transform-regenerator/test/fixtures/arrow-function-inside-body/options.json

```diff
@@ -0,0 +1,5 @@
+{
+  "plugins": [
+    "transform-regenerator"
```

Comment:
I'm trying to add a ES2015 presets ([like here](https://github.com/yavorsky/babel/blob/202ca7fe1d91885515ad8ecd7b2fdc256e2d74d1/packages/babel-plugin-transform-regenerator/test/fixtures/regression/options.json#L3)), currently not working for me